cenpy/products.py

```diff
@@ -1,7 +1,10 @@
+from .utilities import _replace_missing as replace_missing_func 
```

Comment:
why are these private functions being made public here?

Reply:
They shouldn't be. I'll fix that in a few.

Reply:
I addressed this in my most recent commit.